src/components/OpenFileDialog/OwnerFilter.js

```diff
@@ -0,0 +1,24 @@
+import i18n from '@dhis2/d2-i18n'
+import { SingleSelect, SingleSelectOption } from '@dhis2/ui'
+import PropTypes from 'prop-types'
+import React from 'react'
+
+export const OwnerFilter = ({ selected, onChange }) => (
+    <SingleSelect
+        selected={selected}
+        onChange={({ selected }) => onChange(selected)}
+        prefix={i18n.t('Created by')}
+        dense
+    >
+        <SingleSelectOption label={i18n.t('Anyone')} value="all" />
```

Comment:
This will be hard to translate in languages that has a different flow to the sentence "Created by anyone". Can we change it to become sort of a two part thing, e.g. "Creator: Anyone" instead of a sentence? So they can be translated separately.  @cooper-joe

Reply:
Added a TODO comment.